c7n/resources/bedrock.py

```diff
@@ -121,3 +122,208 @@ def process(self, resources):
+        permission_prefix = 'bedrock'
+
+    source_mapping = {
+        'describe': DescribeBedrockCustomizationJob
+    }
+
+
+@BedrockModelCustomizationJob.filter_registry.register('kms-key')
+class BedrockCustomizationJobsKmsFilter(KmsRelatedFilter):
+    """
+
+    Filter bedrock customization jobs by its associcated kms key
+    and optionally the aliasname of the kms key by using 'c7n:AliasName'
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: bedrock-customization-job-kms-key-filter
+            resource: aws.model-customization-job
+            filters:
+              - type: kms-key
+                key: c7n:AliasName
+                value: alias/aws/bedrock
+
+    """
+    RelatedIdsExpression = 'outputModelKmsKeyArn'
+
+
+@BedrockModelCustomizationJob.action_registry.register('tag')
+class TagModelCustomizationJob(Tag):
+    """Create tags on Bedrock model customization jobs
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+            - name: bedrock-model-customization-job-tag
+              resource: aws.model-customization-job
+              actions:
+                - type: tag
+                  key: test
+                  value: something
+    """
+    permissions = ('bedrock:TagResource',)
+
+    def process_resource_set(self, client, resources, new_tags):
+        tags = [{'key': item['Key'], 'value': item['Value']} for item in new_tags]
+        for r in resources:
+            client.tag_resource(resourceARN=r["jobArn"], tags=tags)
+
+
+@BedrockModelCustomizationJob.action_registry.register('remove-tag')
+class RemoveTagModelCustomizationJob(RemoveTag):
+    """Remove tags from Bedrock model customization jobs
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+            - name: bedrock-model-customization-job-remove-tag
+              resource: aws.model-customization-job
+              actions:
+                - type: remove-tag
+                  tags: ["tag-key"]
+    """
+    permissions = ('bedrock:UntagResource',)
+
+    def process_resource_set(self, client, resources, tags):
+        for r in resources:
+            client.untag_resource(resourceARN=r['jobArn'], tagKeys=tags)
+
+
+@BedrockModelCustomizationJob.action_registry.register('stop')
+class StopCustomizationJob(BaseAction):
+    """Stop model customization job
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+            - name: bedrock-model-customization-untagged-stop
+              resource: aws.model-customization-job
+              filters:
+                - tag:Owner: absent
+              actions:
+                - type: stop
+
+    """
+    schema = type_schema('stop')
+    permissions = ('bedrock:StopModelCustomizationJob',)
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('bedrock')
+        for r in resources:
+            client.stop_model_customization_job(jobIdentifier=r['jobArn'])
```

Comment:
this probably needs a catch around state, ie does stopping a stopped job result in error?

Reply:
Thanks for the feedback @kapilt . I did try to run the stop operation in loop and on already stopped jobs. Apparently, it doesn't trigger any exceptions. 
```
>>> client = boto3.client('bedrock', 'us-east-1')
>>> response = client.stop_model_customization_job(jobIdentifier='arn:aws:bedrock:us-east-1:644160558196:model-customization-job/cohere.command-text-v14:7:4k/kktbyp9iwhsw')
>>> response = client.stop_model_customization_job(jobIdentifier='arn:aws:bedrock:us-east-1:644160558196:model-customization-job/cohere.command-text-v14:7:4k/kktbyp9iwhsw')
>>> response = client.stop_model_customization_job(jobIdentifier='arn:aws:bedrock:us-east-1:644160558196:model-customization-job/cohere.command-text-v14:7:4k/kktbyp9iwhsw')
```
Interestingly, I also read the doc re: [exceptions](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_StopModelCustomizationJob.html), and found a `ConflictException` but wasn't able to create a scenario that would trigger it